pgmoon/json.moon

```diff
@@ -0,0 +1,10 @@
+encode_json = (tbl) ->
+  json = require "cjson"
+  enc = json.encode tbl
+  "'#{(enc\gsub "'", "''")}'"
```

Comment:
can we use `escape_literal` here so all escaping is consolidated through same path